cmd/daemon/main.go

```diff
@@ -559,6 +559,10 @@ func main() {
 
 	pb.RegisterDaemonServer(s, rpc)
 	meshpb.RegisterMeshnetServer(s, meshService)
+
+	consentChecker := daemon.NewConsentChecker(fsystem, defaultAPI, authChecker)
```

Comment:
NOTE: the class is probably not needed, check if it won't be annoying to get rid of it and just have a func